zboxcore/sdk/filerefsworker.go

```diff
@@ -82,7 +85,10 @@ func (o *ObjectTreeRequest) GetRefs() (*ObjectTreeResult, error) {
 			hashRefsMap[hash] = oTreeResponse.oTResult
 		}
 	}
-
+	
+	if zboxutil.MajorStatusCode(oTreeResponseStatusCodes) == http.StatusBadRequest { //It should be http.StatusNotFound, but the blobber is returning 400 if the file not found
```

Comment:
Have you checked the responsed message, blobber should return 400 status code along with error messages. We need we need to what happened, 400 could be returned with different errors, and we should only return empty `ObjectTreeResult{}` when the error message indicates 'no record found' or similar error. Otherwise, we should return the major error.

Reply:
Yes, The response message from the blobber is `"400 Bad Request"`.

Reply:
I checked the blobber endpoint `/v1/file/refs/{allocation}`, it should give error message. Anyway, I had a test to call the `GetRefs()` with none exist remote path, it will does show the `invalid path` error message. The main issue that you don't see the error message is because the `getFileRefs` method ignored it. I'll some commits, and you can test with my commits to see the error message returned from blobbers.

Reply:
@peterlimg  Yes, I have corrected now using the error msg (from your commit).